src/components/TestCrash/index.native.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import TestToolRow from '@components/TestToolRow';
+import useLocalize from '@hooks/useLocalize';
+import testCrash from '@libs/testCrash';
+import firebase from '../../../firebase.json';
+
+/**
+ * Adds a button in native dev builds to test the crashlytics integration with user info.
+ */
+function TestCrash() {
+    const {translate} = useLocalize();
+
+    const isCrashlyticsDebugEnabled = firebase?.['react-native']?.crashlytics_debug_enabled ?? false;
+
+    const toolRowTitle = translate('initialSettingsPage.troubleshoot.testCrash');
+
+    return (
+        <View>
+            {isCrashlyticsDebugEnabled ? (
```

Comment:
Don't we want this button to show on staging?

Reply:
>This will show in staging if crashlytics_debug_enabled is true.

So we don't need to set that in `firebase.json` for staging?

Reply:
>I think we want to show this option only when a developer manually enables the crashlytics_debug_enabled flag. Do let me know your thoughts.

On staging how are we going to enable it?

Reply:
I am not sure, it's probably supposed to be tested on AdHoc build or production? @roryabraham

Reply:
We want this PR to work for both staging and production, but not for dev or AdHoc builds by default

Reply:
@ShridharGoel, the button will show for both staging and production, right? I thought the change to `firebase.json` would only apply to dev

Reply:
>The user ID changes will work for staging and production. The testing button would show only when firebase.json is updated manually.

Sorry i am confused here, then why not remove the testing button after confirming the data is being sent to firebase is correct? if it will only be  visible only on dev (which i assume is the only way to update firebase.json)